profiles/testbot.lua

```diff
@@ -19,6 +19,14 @@ function setup()
       use_turn_restrictions         = true
     },
 
+    classes = {[0] = "motorway", [1] = "toll"},
+
+    excludable = {
+        [1] = {["motorway"] = true},
+        [2] = {["toll"] = true},
+        [3] = {["motorway"] = true, ["toll"] = true}
```

Comment:
Turns out the `[1] = "foo", [2] = "bar"` syntax is the same as `{'foo', 'bar'}`. In the other case I would like to make these Sets so you can have duplicate keys in there and order does not matter. I could have used the wrapper but didn't want to include another module in `testbot.lua` just for one line.